### DISCUSSION TITLE
Region wise regularization

### DISCUSSION THREAD

**Discussion by makeabhishek** [gimli-org/pyGIMLi#740] [Q&A]
Hi, 
I'm trying to understand the region wise regularization. I tried the example given https://www.pygimli.org/_tutorials_auto/3_inversion/plot_8-regionWise.html. However, I'm not getting what is going on at the back. What's the math when we implement  `mgr.inv.setRegularization(3, single=True, , limits = [1/9000, 1/100])` ?

Is there any reference available for Region-wise regularization?

**Comment by makeabhishek** [comment]
What is the best way to identify the range of limit. I was trying different limit for a circular body with outer ring. But if range is low than the rays do not penetrate in the inner region.
![image](https://github.com/gimli-org/gimli/assets/47937684/c371850f-6cfa-4436-8cfd-604c70cfdf1e)

**Reply by makeabhishek** [reply]
Hi,
May I ask if there is any reference/research article available?

**Reply by halbmy** [reply]
The limits of the physical properties at hand can only be defined by somebody who knows the object of study, e.g. from petrophysical measurements. This is clearly out of scope for pyGIMLi

**Comment by halbmy** [comment]
The math behind is described by Günther (2004). The model transformation is

![image](https://github.com/user-attachments/assets/7590a8d2-2535-4e9a-9a04-af130c58c2b1)

also published by Kim & Kim (2011).

References
* Günther, T., 2004. Inversion Methods and Resolution Analysis for the 2D/3D Reconstruction of Resistivity Structures from DC Measurements, PhD thesis, Freiberg University of Mining and Technology, available from http://nbn-resolving.de/urn:nbn:de:
swb:105-4152277
* Kim, H.J. & Kim, Y., 2011. A unified transformation function for lower and upper bounding constraints on model parameters in electrical and electromagnetic inversion, J. Geophys. Eng., 8, 21–26.
